db/migrate/20190206213630_add_icon_column_to_portfolioitem.rb

```diff
@@ -0,0 +1,5 @@
+class AddIconColumnToPortfolioitem < ActiveRecord::Migration[5.2]
+  def change
+    add_column :portfolio_items, :service_offering_icon_id, :bigint
```

Comment:
I'm wondering if this should be renamed to `service_offering_icon_ref` to stay consistent with the the other `_ref` attributes we're using. Thoughts?

Reply:
I agree, I like that a bit better.